main/templates/main/pages/submission.html

```diff
@@ -32,7 +32,7 @@
     <div>
       <script type="text/javascript">
         let a = '{{ entries | escapejs}}';
-        var mapbox_user_name = "{{mapbox_user_name}}";
+        let mapbox_user_name = "{{mapbox_user_name}}";
```

Comment:
hmm.. this would display our mabox_user_name to all users - do we need to include?

Reply:
i'm not totally sure about this one.. we've been using this line since pretty much the beginning. unless you mean the change from var to let? i don't know that it changes things, but i'm not totally clear on this aspect of things (imagine if we were taught javascript at princeton...)